src/object/mod.rs

```diff
@@ -135,8 +135,9 @@ mod traits {
     use crate::context::Context;
     use crate::handle::{Handle, Managed, Root};
     use crate::result::{NeonResult, Throw};
+    use crate::types::function::CallOptions;
     use crate::types::utf8::Utf8;
-    use crate::types::{build, JsUndefined, JsValue, Value};
+    use crate::types::{build, JsFunction, JsUndefined, JsValue, Value};
```

Comment:
Nit, these additions are throwing unused warnings on the legacy backend. Probably not a big deal because that's getting removed very soon.

Reply:
death to legacy…